tests/update_workflow.go

```diff
@@ -3506,26 +3364,20 @@ func (s *FunctionalSuite) TestUpdateWorkflow_SpeculativeWorkflowTask_Heartbeat()
 	}()
 
 	// Heartbeat from workflow.
-	res, err := poller.PollAndProcessWorkflowTask(WithRetries(1), WithForceNewWorkflowTask)
+	res, err := poller.PollAndProcessWorkflowTask(WithoutRetries, WithForceNewWorkflowTask)
 	s.NoError(err)
 	heartbeatResp := res.NewTask
 
 	// Reject update from workflow.
-	updateResp, err := poller.HandlePartialWorkflowTask(heartbeatResp.GetWorkflowTask(), true)
+	updateResp, err := poller.HandlePartialWorkflowTask(heartbeatResp.GetWorkflowTask(), false)
```

Comment:
side note: I wish Go had better enums; this `false` here tells me nothing. (I looked into Goland to get more details)

Reply:
Luckly IDE shows it.